<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>762</width>
     <height>520</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>762</width>
     <height>520</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>762</width>
     <height>520</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(47, 92, 147, 255), stop:1 rgba(26, 0, 31, 255));
border-radius:20;
border-style: solid;
border-width: 3px;
border-color: rgb(85, 255, 255);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="7,1">
    <item>
     <widget class="QWidget" name="videoWidget" native="true">
      <property name="styleSheet">
       <string notr="true">border: none;
border-radius: 0;
background-color: black;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="styleSheet">
       <string notr="true">background-color: black;
border-radius:20;
border-style: solid;
border-width: 3px;
border: none;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="b_abrir">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(47, 92, 147, 255), stop:1 rgba(26, 0, 31, 255));
border-style: solid;
border-radius: 20;
font:75 12pt &quot;Myanmar Text&quot;;
font: bold;
color: rgb(85, 255, 255);
}
QPushButton:hover{
background-color:rgba(3, 239, 254, 0.4);
font:75 12pt &quot;Myanmar Text&quot;;
font: bold;
color: rgb(3,239,254);
border-radius:20;
}</string>
         </property>
         <property name="text">
          <string>Abrir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_reproducir">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(47, 92, 147, 255), stop:1 rgba(26, 0, 31, 255));
border-style: solid;
border-radius: 20;
font:75 12pt &quot;Myanmar Text&quot;;
font: bold;
color: rgb(85, 255, 255);
}
QPushButton:hover{
background-color:rgba(3, 239, 254, 0.4);
font:75 12pt &quot;Myanmar Text&quot;;
font: bold;
color: rgb(3,239,254);
border-radius:20;
}</string>
         </property>
         <property name="text">
          <string>Reproducir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_pausa">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(47, 92, 147, 255), stop:1 rgba(26, 0, 31, 255));
border-style: solid;
border-radius: 20;
font:75 12pt &quot;Myanmar Text&quot;;
font: bold;
color: rgb(85, 255, 255);
}
QPushButton:hover{
background-color:rgba(3, 239, 254, 0.4);
font:75 12pt &quot;Myanmar Text&quot;;
font: bold;
color: rgb(3,239,254);
border-radius:20;
}</string>
         </property>
         <property name="text">
          <string>Pausa</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_detener">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(47, 92, 147, 255), stop:1 rgba(26, 0, 31, 255));
border-style: solid;
border-radius: 20;
font:75 12pt &quot;Myanmar Text&quot;;
font: bold;
color: rgb(85, 255, 255);
}
QPushButton:hover{
background-color:rgba(3, 239, 254, 0.4);
font:75 12pt &quot;Myanmar Text&quot;;
font: bold;
color: rgb(3,239,254);
border-radius:20;
}</string>
         </property>
         <property name="text">
          <string>Detener</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_regresar">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(47, 92, 147, 255), stop:1 rgba(26, 0, 31, 255));
border-style: solid;
border-radius: 20;
font:75 12pt &quot;Myanmar Text&quot;;
font: bold;
color: rgb(85, 255, 255);
}
QPushButton:hover{
background-color:rgba(3, 239, 254, 0.4);
font:75 12pt &quot;Myanmar Text&quot;;
font: bold;
color: rgb(3,239,254);
border-radius:20;
}</string>
         </property>
         <property name="text">
          <string>Regresar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
